### ISSUE TITLE
HexTags and VIP Chat by r1ko bugs out

### ISSUE THREAD

**Issue by nDy1k37** [Hexer10/HexTags#93]
![image](https://user-images.githubusercontent.com/73367624/222579301-6ca2027e-0961-4a54-9145-4d715b6b441a.png)

So my chat prefixes has double spaces between HexTags "[OWNER]" and the VIP Chat Prefix "VIP GOLD", any help?

**Comment by crashzk** [comment]
@oribiler config for hextags and VIP Core chat/tag modules, send them all, so we can verify.

**Comment by nDy1k37** [comment]
https://mega.nz/file/Uc9xkATT#bQDW2ipwanonpcqDKEBcPB6tFn7Osar08ilsm4_PkcY here they are, is this enough?

**Comment by crashzk** [comment]
@oribiler test this setting in **`hextags.cfg`**:
```
"HexTags"
{
	"@Owner"
	{
		"ScoreTag" "[OWNER]" //The scoreboard-tag
		"ChatTag"	"{grey}[{darkred}Owner{grey}]" //The chat-tag
		"ChatColor" "{darkred}" //The chat color
		"NameColor" "{darkred}" //The name color
	}
    
	"@Co-Owner"
	{
		"ScoreTag" "[CO-OWNER]" //The scoreboard-tag
		"ChatTag"	"{grey}[{red}Co-Owner{grey}]" //The chat-tag
		"ChatColor" "{red}" //The chat color
		"NameColor" "{red}" //The name color
	}

	"@Administrator" 
	{
		"ScoreTag" "[ADMIN]" //The scoreboard-tag
		"ChatTag"	"{grey}[{green}Administrator{grey}]" //The chat-tag
		"ChatColor" "{green}" //The chat color
		"NameColor" "{green}" //The name color
	}

	"@Moderator" 
	{
		"ScoreTag" "[MOD]" //The scoreboard-tag
		"ChatTag"	"{grey}[{orchid}Moderator{grey}]" //The chat-tag
		"ChatColor" "{orchid}" //The chat color
		"NameColor" "{orchid}" //The name color
	}

	"@Helper" 
	{
		"ScoreTag" "[HELPER]" //The scoreboard-tag
		"ChatTag"	"{grey}[{yellow}Helper{grey}]" //The chat-tag
		"ChatColor" "{yellow}" //The chat color
		"NameColor" "{yellow}" //The name color

	}
	
	"STEAM_1:0:568409728" 
	{
		"ScoreTag" "" //The scoreboard-tag
		"ChatTag"	"{grey}[{BLUE}Cioara{grey}]" //The chat-tag
		"ChatColor" "{BLUE}" //The chat color
		"NameColor" "{blue}" //The name color
	}
}
```

**Comment by nDy1k37** [comment]
It would remove the spaces between rank and name for non-vips

**Comment by crashzk** [comment]
> It would remove the spaces between rank and name for non-vips

Wouldn't that be the case? Or I misunderstood your problem, explain anything again, more details I try to reproduce here.

**Comment by nDy1k37** [comment]
Well the problem is, I have 2 spaces between VIP Prefix and HexTags Prefix and I want to remove them, but if I remove the space from the hextags config, my chat would look like [OWNER]oribiler, without vip-status.

**Comment by crashzk** [comment]
> Well the problem is, I have 2 spaces between VIP Prefix and HexTags Prefix and I want to remove them, but if I remove the space from the hextags config, my chat would look like [OWNER]oribiler, without vip-status.

Seeing the screenshot you sent above, the spacing is fine as you don't want to remove the spacing in the hextags.

Try removing the spacing from the hextags, and in the VIP Chat plugin try including the space before the tag, see if that works for you.

**Comment by nDy1k37** [comment]
I removed it and it looks broken, it's a problem from ChatProcessor or VIP-Chat I believe

**Comment by crashzk** [comment]
> I removed it and it looks broken, it's a problem from ChatProcessor or VIP-Chat I believe

In this case, only authors of these plugins can help, try reporting to them and see if there is any response. Also try to isolate in which one the problem occurs, deactivate VIP-Chat and see if it still persists, or only when you activate the VIP-Chat plugin.

**Comment by nDy1k37** [comment]
If I remove VIP-Chat, the problem resolves, but now I discover that HexTags and Shop Prefixes are buggy too and double spaces, so it's not from VIP-Chat, it's probably because of chat-processor.

**Comment by crashzk** [comment]
@oribiler test, maybe it can solve it.

https://github.com/crashzk/chat-processor/tree/fix-ColorsChat-ChatGOTV

In any case, if it continues, I recommend reporting it to chat-processor issues.

**Comment by nDy1k37** [comment]
Nope, it doesn't fix anything, still buggy, let me report it to chat-processor then.
